src/awaitVercelDeployment.ts

```diff
@@ -10,13 +10,17 @@ import { VercelDeployment } from './types/VercelDeployment';
  * @param timeout Duration (in seconds) until we'll await for.
  *  When the timeout is reached, the Promise is rejected (the action will fail).
  */
-const awaitVercelDeployment = (baseUrl: string, timeout: number): Promise<VercelDeployment> => {
+const awaitVercelDeployment = (baseUrl: string, teamId: string | undefined, timeout: number): Promise<VercelDeployment> => {
   return new Promise(async (resolve, reject) => {
     let deployment: VercelDeployment = {};
     const timeoutTime = new Date().getTime() + timeout;
 
     while (new Date().getTime() < timeoutTime) {
-      deployment = await fetch(`${VERCEL_BASE_API_ENDPOINT}/v11/now/deployments/get?url=${baseUrl}`, {
+      let url = `${VERCEL_BASE_API_ENDPOINT}/v13/deployments/${baseUrl}`;
```

Comment:
I did not find any documentation regarding the old v11 api, and seems the `team id` querystring does not work on v11 api. I also did a brief search and did not find the api changelogs.

Reply:
https://vercel.com/docs/rest-api#endpoints/deployments/list-deployment-builds (v11)
https://vercel.com/docs/rest-api#endpoints/deployments/get-a-deployment-by-id-or-url (v13)

Both support the `teamId` parameter, but v13 changes the format of the response.

To avoid breaking things, and changing types, I would prefer to keep using v11 for now.